src/modules/app/StateType.js

```diff
@@ -45,6 +45,32 @@ export type CategoryRouteStateType = {|
   +code: ErrorCodeType
 |}
 
+export type PoiRouteConfigType = {|
+  +path: ?string, // path is null for the poi-lists route
+  +language: string,
+  +city: string
+|}
+
+export type PoiRouteStateType = {|
+  +status: 'ready',
+  ...PoiRouteConfigType,
+  +models: $ReadOnlyArray<PoiModel>,
+  +allAvailableLanguages: $ReadOnlyMap<string, ?string> // including the current content language
```

Comment:
Why do we need `?string` here?

Reply:
I saw it in other RouteStateTypes so i added it here too. Not sure if it is needed.

Reply:
For events we have it because of this snippet:

```
      14│   const getEventRoute = (): EventRouteStateType => {
      15│     if (!path) {
 [1]  16│       const allAvailableLanguages = new Map(cityLanguages.map(lng => [lng.code, null]))
      17│       return { status: 'ready', path: null, models: events, allAvailableLanguages, language, city }
      18│     }
      19│     const event: ?EventModel = events.find(event => event.path === path)
```

Reply:
For the events, if a path is falsy, we interpret it as the `overview` / `list` view of all events.

Reply:
@TechPiranja can you add a note to the StateType definitions.

```
type LanguageKey = string
type Path = ?string // Path can be falsy if the current displayed view is a list of events or POIs
type AllAvailableLanguagesType = $ReadOnlyMap<LanguageKey, Path>
```

Then reuse these definitions in the StateType:
```
+allAvailableLanguages: AllAvailableLanguagesType
```